ice-cream-the-fat-cat/ice-cream-backend
# Added first connection to MongoDB

## Description

Added a simple connection to the dev MongoDB Atlas database.

## Closes issue(s)

#5 

## How to test / reproduce

After the build is done, if you check the endpoint `/icecream-test` from your browser, you should get a list of databases in our MongoDB Atlas account

## Screenshots

<img width="1168" alt="スクリーンショット 2021-08-25 16 33 03" src="https://user-images.githubusercontent.com/770480/130746808-c81723a2-d2a7-40a6-b676-cfb023afad02.png">

## Changes include


## Checklist


## Other comments
